assets/airports/eddh.json

```diff
@@ -1,844 +1,1435 @@
+            ]
         },
         "NOLGO": {
             "icao": "NOLGO",
             "name": "Nolgo",
+            "suffix": {"05":"05", "15":"15", "23":"23", "33":"33"},
             "entryPoints": {
-                "P605": ["DLE", ["MADUB", "A280"], ["AGATI", "A200"]]
+                "NOLGO": []
             },
-            "body": [["NOLGO", "A100|S250"]],
+            "body": ["NOLGO"],
             "rwy": {
-                "05": [["DH230", "S220"], "DH235", "DH255", "DH256", "DH257", "DH025", "ANEXI"],
-                "15": ["DH406", ["DH412", "S220"], "DH416", "DH455", "DH456", "DH457", "DH458"],
-                "23": [["DH611", "S220"], "DH615", "DH655", "DH656", "DH657", "DH658"],
-                "33": ["DH832", ["DH835", "S220"], "DH854", "DH855", "DH856", "DH857", "DH858"]
+                "05": [["DH230", "S220"], "DH231", "DH232", "DH233", "DH234", ["DH235", "S220"], "DH255", "DH256", "DH257", "DH025", ["ANEXI", "S170+"]],
+                "15": ["DH406", "DH407", "DH408", "DH409", ["DH412", "S220"], "DH413", "DH414", ["DH416", "S220"], "DH455", "DH456", "DH457", "DH458", ["DEVOD", "S170+"]],
+                "23": [["DH611", "S220"], "DH612", "DH613", "DH614", ["DH615", "S220"], "DH655", "DH656", "DH657", "DH658", ["PISAS", "S170+"]],
+                "33": ["DH832", "DH833", "DH834", ["DH835", "S220"], "DH854", "DH855", "DH856", "DH857", "DH858", ["SOSAX", "S170+"]]
             },
-            "draw": [[]]
+            "draw": [
+                ["NOLGO*", "DH230", "DH231", "DH232", "DH233", "DH234", "DH235", "DH255", "DH256", "DH257", "DH025", "ANEXI"],
+                ["NOLGO*", "DH406", "DH407", "DH408", "DH409", "DH412", "DH413", "DH414", "DH416", "DH455", "DH456", "DH457", "DH458", "DEVOD"],
+                ["NOLGO*", "DH611", "DH612", "DH613", "DH614", "DH615", "DH655", "DH656", "DH657", "DH658", "PISAS"],
+                ["NOLGO*", "DH832", "DH833", "DH834", "DH835", "DH854", "DH855", "DH856", "DH857", "DH858", "SOSAX"]
+            ]
         },
         "RARUP": {
             "icao": "RARUP",
             "name": "Rarup",
+            "suffix": {"05":"05", "15":"15", "23":"23", "33":"33"},
             "entryPoints": {
-                "M748": ["BKD", ["BUMIL", "A230"], ["ABMAL", "A150"]]
+                "RARUP": []
             },
-            "body": [["RARUP", "A100|S250"]],
+            "body": ["RARUP"],
             "rwy": {
-                "05": [["DH230", "S220"], "DH235", "DH255", "DH256", "DH257", "DH025", "ANEXI"],
-                "15": [["DH410", "S220"], "DH412", "DH416", "DH455", "DH456", "DH457", "DH458"],
-                "23": [["DH600", "S220"], "DH611", "DH615", "DH655", "DH656", "DH657", "DH658"],
-                "33": [["DH829", "S220"], "DH830", "DH835", "DH854", "DH855", "DH856", "DH857", "DH858"]
+                "05": [["DH230", "S220"], "DH231", "DH232", "DH233", "DH234", ["DH235", "S220"], "DH255", "DH256", "DH257", "DH025", ["ANEXI", "S170+"]],
```

Comment:
Where you have `DH025`, I show `DH258`, though they appear to have the same coordinates.